addon/helpers/changeset.js

```diff
@@ -6,20 +6,20 @@ import isPromise from 'ember-changeset/utils/is-promise';
 
 const { Helper: { helper } } = Ember;
 
-export function changeset([model, validationMap]) {
+export function changeset([model, validationMap], options) {
   if (isObject(validationMap)) {
     if (isPromise(model)) {
-      return model.then((resolved) => new Changeset(resolved, lookupValidator(validationMap), validationMap));
+      return model.then((resolved) => new Changeset(resolved, lookupValidator(validationMap), validationMap, options));
     }
 
-    return new Changeset(model, lookupValidator(validationMap), validationMap);
+    return new Changeset(model, lookupValidator(validationMap), validationMap, options);
   }
 
   if (isPromise(model)) {
-    return model.then((resolved) => new Changeset(resolved, validationMap));
+    return model.then((resolved) => new Changeset(resolved, validationMap, options));
```

Comment:
For the record, I have the signature for `Changeset` wrong here. It should be `new Changeset(resolved, validationMap, {}, options)`, as it is in #147. So yeah, definitely go with that one. :)